docs/features/configuration.md

```diff
@@ -45,6 +45,9 @@ Some companies disallow the usage of Docker Hub, but you can override `*.image`
 > **localstack.container.image = localstack/localstack**  
 > Used by LocalStack
 
+Other possibility is to set up a registry mirror in your environment, so that all images are pulled from there and not directly from Docker Hub.
```

Comment:
```suggestion
Another possibility is to set up a registry mirror in your environment so that all images are pulled from there and not directly from Docker Hub.
```